modules/ROOT/pages/browser-support.adoc

```diff
@@ -14,6 +14,13 @@ Use Anypoint Platform with the latest version of any supported browser:
 
 The Anypoint Platform browser support follows the Salesforce Lightning Experience policy.
 
+== Exceptions to Browser Support Policy
+
+API Community Manager and Partner Manager 2.0 each modify the browser support policy:
+
+* xref:api-community-manager::data-sheets.adoc[API Community Manager]
+* xref:release-notes::partner-manager/anypoint-partner-manager-2.0-release-notes#browser-compatibility.adoc[Partner Manager 2.0]
+
 == Unsupported Browsers
 
 Internet Explorer versions 11 and earlier are not supported by MuleSoft.
```

Comment:
MuleSoft > Mule runtime engine? Anypoint Platform?

Reply:
Great catch! Anypoint Platform, thank you.

Reply:
I just deleted the prepositional phrase at the end. Given the following sentence, it seemed unnecessary.